aerospike-core/src/policy/concurrency.rs

```diff
@@ -27,15 +27,4 @@ pub enum Concurrency {
     /// extremely large batch sizes because each node can process the request immediately. The
     /// downside is extra threads will need to be created (or takedn from a thread pool).
     Parallel,
-
-    /// Issue up to N commands in parallel threads. When a request completes, a new request
-    /// will be issued until all threads are complete. This mode prevents too many parallel threads
-    /// being created for large cluster implementations. The downside is extra threads will still
-    /// need to be created (or taken from a thread pool).
-    ///
-    /// E.g. if there are 16 nodes/namespace combinations requested and concurrency is set to
-    /// `MaxThreads(8)`, then batch requests will be made for 8 node/namespace combinations in
-    /// parallel threads. When a request completes, a new request will be issued until all 16
-    /// requests are complete.
-    MaxThreads(usize),
```

Comment:
This is gone... threadpool size is best handled by the async runtime like tokio.

Instead Parallel and Sequential are _both_ parallel, however Parallel uses multiple tasks (and thus potentially threads) and Sequential just uses join_all to drive them all from the current thread.